### ISSUE TITLE
Third Column in Theme

### ISSUE THREAD

**Issue by willykaram** [NYC-Camp/website#251]
We absolutely need a 3rd column on the web site. If we can possibly add it to this theme that would really help us for many purposes, including:
- **Most importantly**, displaying sponsor information
- Displaying total number of registered users
- Users Login/Registration Form
- Conference and Events Registration Forms
- Displaying shopping cart for purchase of partnerships

I haven't had a chance to look at the theme to see what's involved, but we need this issue resolved pretty badly

**Comment by willykaram** [comment]
BTW, this is marked as a blocker and critical because it's impeding us form doing some very important things on the site.

**Comment by willykaram** [comment]
I am starting work on this and will have it done for Sat. Just commenting in order to keep the team in the loop. I didn't see anyone else take a look at/assign this to themselves.

**Comment by esod** [comment]
Thanks for the update. Don't put this on live until it's been QA'd.

**Comment by willykaram** [comment]
Yes Eric. As discussed last week, (i) the prior theming was done overnight
and on an urgent basis as part of getting the session scheduling working and (ii) there is a theming branch where other theming work I've done has gone.

**Comment by esod** [comment]
Great. sounds good.

If you're online, will you merge my pull request, https://github.com/NYC-Camp/website/pull/256. I promise it's fine and it closes the loop on a problem I've been working for days.

Also, once the module and configuration are out, I want to follow up on Forest's comment in https://groups.drupal.org/node/398153 with something like this:

"I'd like to add that you can quickly view your Sessions and your Registrations by going to the menu items in My Profile on the main menu across the top."

or something to that effect. 

Thanks
